src/System.ServiceModel.Primitives/pkg/System.ServiceModel.Primitives.pkgproj

```diff
@@ -18,6 +18,7 @@
     <InboxOnTargetFramework Include="MonoTouch10" />
     <InboxOnTargetFramework Include="net45" />
     <InboxOnTargetFramework Include="win8" />
+    <InboxOnTargetFramework Include="wp8" />
```

Comment:
@ericstj 
This wasn't in the S.SM.Primitives.nuproj, how did you know it is needed?